source/configuration_manual/stats/openmetrics.rst

```diff
@@ -0,0 +1,74 @@
+===================================
+OpenMetrics exporter for statistics
+===================================
+
+.. versionadded:: 2.3.11
+
+Basic configuration
+===================
+
+Dovecot has support for `OpenMetrics <https://openmetrics.io/>`_ exposition for statistics.
+This can be enabled by adding following configuration::
```

Comment:
is two :: intentional?..

Reply:
yes

Reply:
shorthand for code-block

Reply:
But it's not used anywhere in the repository now. Should stay consistent.

Reply:
oh, uh, what, I thought I was answering about another thing

Reply:
or I guess it's basically the same

Reply:
But I see it's used in some other places already. So .. I suppose it's okay then. Just will be difficult to grep for them if we need to do things like the idea of converting code-blocks to that other type that allows links.